packages/ckb-sdk-core/examples/sendTransaction.js

```diff
@@ -11,19 +11,21 @@ const bootstrap = async () => {
   const secp256k1Dep = await core.loadSecp256k1Dep() // load the dependencies of secp256k1 algorithm which is used to verify the signature in transaction's witnesses.
 
   /**
-   * genereat address object, who has peroperties like private key, public key, sign method and verify mehtod
-   * - value, the address string
+   * genereat key pair object, who has peroperties as private key, public key, mainnet address, testnet address, sign method and verify mehtod
+   * - mainnetAddress, the address string for the Mainnet
```

Comment:
It's still mixing address with keys. Key pair should have single responsibility of holding private and public key and provide secp256k1 signing. It has nothing to do with addresses.